packages/mapviewer/src/modules/menu/components/advancedTools/ImportCatalogue/ProviderList.vue

```diff
@@ -57,26 +229,125 @@ defineExpose({ goToFirst })
             class="providers-list"
             data-cy="import-provider-list"
         >
+            <template v-for="node in treeData" :key="node.id">
+                <component
+                    :is="node.type === 'group' ? 'div' : 'div'"
```

Comment:
that might be some left-over, but is not relevant anymore. This will create a `<div>` regardless of the state, so you could remove the whole `<component>` thing and replace it with a `<div>`